app/shared/resource-map/UserMarker.js

```diff
@@ -0,0 +1,35 @@
+import L from 'leaflet';
+import shadowUrl from 'leaflet/dist/images/marker-shadow.png';
+import React, { PropTypes } from 'react';
+import { Marker as LeafletMarker } from 'react-leaflet';
+
+import iconRetinaUrl from 'assets/map/usermarker-icon-2x.png';
+import iconUrl from 'assets/map/usermarker-icon.png';
+
+class UserMarker extends React.Component {
+  render() {
+    const icon = L.icon({
+      iconAnchor: [12, 41],
+      iconRetinaUrl,
+      iconSize: [25, 41],
+      iconUrl,
+      popupAnchor: [1, -34],
+      shadowSize: [41, 41],
+      shadowUrl,
+      tooltipAnchor: [16, -28],
+    });
+    return (
+      <LeafletMarker
+        icon={icon}
+        position={[this.props.latitude, this.props.longitude]}
+      />
+    );
+  }
+}
+
+UserMarker.propTypes = {
```

Comment:
- [ ] We could start declaring the propTypes as static propTypes or before a stateless functional component. So let's move towards that whenever we add new files or refactor old ones.